### PR TITLE
Add support for item level classes and 'open in a new tab' feature

### PR DESCRIPTION
**Pull request by breaktag**
I have added these for my own project and thought they might be handy. Firstly I have a requirement that means I need different class names on each of my menu items. Secondly one of my sidebar items is a link to an external URL and I wanted to be able to open it in a new browser tab.

Thanks

### CODE REVIEW

File: resources/views/group.blade.php [hunk 1]
```diff
@@ -1,5 +1,5 @@
 @if($group->shouldShowHeading())
-    <li class="menu-title">{{ $group->getName() }}</li>
+    <li class="header">{{ $group->getName() }}</li>
```
Comment (patrickbrouwers) [inline resources/views/group.blade.php]: This would be a breaking change, please revert.

File: readme.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-# Laravel Sidebar
+# Laravel Sidebar 2.1.1
```
Comment (patrickbrouwers) [inline readme.md]: Remove version number.

**Review by patrickbrouwers — CHANGES_REQUESTED** [review]

**Comment by patrickbrouwers** [comment]
Thanks for your contribution, please check my inline-comments.

**Comment by patrickbrouwers** [comment]
Thanks

**Comment by breaktag** [comment]
No worries

**Pull request merged**